### ISSUE TITLE
subsequent "registerStandardSchemes" calls reset previous calls

### ISSUE THREAD

**Issue by vladimiry** [electron/electron#15943]
Node.js: 10.2.0
Chromium: 66.0.3359.181
Platform: linux (have not tested macos/win32 platforms, might also be affected)
Electron: 3.1.0-beta.2. 

**Expected Behavior**

Option to choose:
1. I can safely call as `electron.protocol.registerStandardSchemes` as many times as I want.
2. Note in the documentation that `electron.protocol.registerStandardSchemes` is supposed to be called only once. 

**Actual behavior**

Subsequent `registerStandardSchemes` calls reset previous calls, as result calls as `window.localStorage` or `window.sessionStorage` end up with the following errors:
- Failed to read the 'localStorage' property from 'Window': Access is denied for this document.
- Failed to read the 'sessionStorage' property from 'Window': Access is denied for this document.

Loading relative resources still works though, see relative image loading case.

**To Reproduce**

This repo https://github.com/vladimiry-forks/electron-bug-register-standard-schemes, this line https://github.com/vladimiry-forks/electron-bug-register-standard-schemes/blob/b127cb5e7090d81aa0fac9323f3843db6cd64019/main.js#L15

**Additional Information**

Workaround is to call `electron.protocol.registerStandardSchemes` once as shown here https://github.com/vladimiry-forks/electron-bug-register-standard-schemes/blob/b127cb5e7090d81aa0fac9323f3843db6cd64019/main.js#L14

![electron-bug-register-standard-schemes](https://user-images.githubusercontent.com/1560781/49425383-7854e580-f7ae-11e8-901b-48af498df564.png)

**Comment by sofianguy** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron v4.2.x or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons. This issue will be closed 7 days from now if there is no response.

Thanks in advance! Your help is appreciated.

**Comment by sofianguy** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.
